### ISSUE TITLE
It cannot load the image :( 

### ISSUE THREAD

**Issue by cuongnv-dev** [muhammadwafa/expo-fast-image#5]

**Comment by mariomurrent-softwaresolutions** [comment]
Do you have any error message or something to get started with?

**Comment by ggepenyan** [comment]
@mariomurrent-softwaresolutions I use Expo and get issue with it. Seems it doesn't work with Expo?

**Comment by ggepenyan** [comment]
The thing is that I'm not able to load webp images on ios 12-13

**Comment by mariomurrent-softwaresolutions** [comment]
It is working with expo, I tested it.
Only webp images or any image?

**Comment by ggepenyan** [comment]
Not able to load webp images on iOS 12-13. Used multiple packages without any help. I'm stuck.

**Comment by mariomurrent-softwaresolutions** [comment]
I guess you need webp support for react native ...
have you tried any other images than webp? e.g. png?

**Comment by ggepenyan** [comment]
Yes, png images are working nice. Also, the issue appears only on ios 13 and below. For the later ios versions and all android devices webp images are working well

**Comment by mariomurrent-softwaresolutions** [comment]
Maybe this helps for older iOS versions: https://github.com/Aleksefo/react-native-webp-format
